pkg/operation/seed/scheduler/gardener_kube_scheduler_bootstrap.go

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scheduler
+
+import (
+	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
+	"github.com/gardener/gardener/pkg/client/kubernetes"
+	"github.com/gardener/gardener/pkg/features"
+	gardenletfeatures "github.com/gardener/gardener/pkg/gardenlet/features"
+	"github.com/gardener/gardener/pkg/operation/botanist/component"
+	"github.com/gardener/gardener/pkg/operation/seed/scheduler/configurator"
+	schedulerconfigv18 "github.com/gardener/gardener/pkg/operation/seed/scheduler/v18"
+	schedulerconfigv19 "github.com/gardener/gardener/pkg/operation/seed/scheduler/v19"
+	schedulerconfigv1alpha2 "k8s.io/kube-scheduler/v18/v1alpha2"
+	schedulerconfigv1beta1 "k8s.io/kube-scheduler/v19/v1beta1"
+
+	"github.com/Masterminds/semver"
+	"github.com/pkg/errors"
+	admissionregistrationv1beta1 "k8s.io/api/admissionregistration/v1beta1"
+	"k8s.io/utils/pointer"
+)
+
+// Bootstrap is used to bootstrap gardener-kube-scheduler in Seed clusters.
+func Bootstrap(
+	c kubernetes.Interface,
+	image string,
+	seedVersion *semver.Version,
+) (component.DeployWaiter, error) {
+	var (
+		config configurator.Configurator
+		err    error
+	)
+	const namespace = "gardener-kube-scheduler"
+
+	if seedVersion == nil {
+		return nil, errors.New("seedVersion is required")
+	}
+
+	switch seedVersion.Minor() {
+	case int64(18):
```

Comment:
What about having some `github.com/Masterminds/semver` constraints and using them here (e.g., like in https://github.com/gardener/gardener/blob/master/pkg/operation/botanist/controlplane/kubescheduler/kube_scheduler.go#L465-L503)?